liquidity/liquidity.go

```diff
@@ -156,9 +176,17 @@ type Config struct {
 	// ListLoopOut returns all of the loop our swaps stored on disk.
 	ListLoopOut func() ([]*loopdb.LoopOut, error)
 
+	// GetLoopOut returns a single loop out swap based on the provided swap
+	// hash.
+	GetLoopOut func(hash lntypes.Hash) (*loopdb.LoopOut, error)
+
 	// ListLoopIn returns all of the loop in swaps stored on disk.
 	ListLoopIn func() ([]*loopdb.LoopIn, error)
 
+	// GetLoopIn returns a single loop in swap based on the provided swap
+	// hash.
+	GetLoopIn func(hash lntypes.Hash) (*loopdb.LoopIn, error)
```

Comment:
GetLoopIn is unused, I think we can lower the diff of this PR if loop in amount backoff is not in the scope here.

Reply:
Yes amount back-off is only meant to be applied on the payment part of a loop-out. Any reason we should keep it for consistency?

Reply:
Given it's not used, I think it's safe to remove.